### PR TITLE
pythonPackages.python-louvain: init at 0.15

### PR DESCRIPTION
**Pull request by erictapen**
###### Motivation for this change

I'm not sure about the [license](https://github.com/taynaud/python-louvain/blob/master/LICENSE) here.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 127818` at 4be2261d run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.python-louvain</li>
    <li>python39Packages.python-louvain</li>
  </ul>
</details>

**Comment by fabaff** [comment]
> I'm not sure about the license here.

Looks like [BSD3](https://opensource.org/licenses/BSD-3-Clause).

**Comment by SuperSandro2000** [comment]
> > I'm not sure about the license here.
> 
> Looks like [BSD3](https://opensource.org/licenses/BSD-3-Clause).

https://github.com/taynaud/python-louvain/blob/master/setup.py#L9

File: pkgs/development/python-modules/python-louvain/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  pname = "python-louvain";
+  version = "0.15";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-KoVu374plSpgpVOKhLt4zKGPaISoi5Ml6FoRyN1JF+s=";
+  };
+
+  propagatedBuildInputs = [ networkx numpy ];
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/python-louvain/default.nix:17]:
```suggestion

  # no tests
  doCheck = false;

```

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 127818` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>python38Packages.python-louvain</li>
<li>python39Packages.python-louvain</li>
</ul>
</details>

**Comment by erictapen** [comment]
> > > I'm not sure about the license here.
> > 
> > 
> > Looks like [BSD3](https://opensource.org/licenses/BSD-3-Clause).
> 
> https://github.com/taynaud/python-louvain/blob/master/setup.py#L9

So I guess I can keep `licenses.bsd3`?

**Comment by SuperSandro2000** [comment]
> So I guess I can keep `licenses.bsd3`?

yes

**Comment by erictapen** [comment]
@SuperSandro2000 Why did you remove the tests? They build fine for me.

**Comment by SuperSandro2000** [comment]
I thought there are no tests but they brewed something custom.

**Comment by erictapen** [comment]
Even if it had none, wouldn't it be better to run `checkPhase` nonetheless? Upstream could add them later. Anyway, I'd like to enable them in https://github.com/NixOS/nixpkgs/pull/127926.

**Comment by SuperSandro2000** [comment]
> Even if it had none, wouldn't it be better to run checkPhase nonetheless?

No because it is unclear what then happens.

**Pull request merged**